### DISCUSSION TITLE
Wordpress

### DISCUSSION THREAD

**Discussion by ikiteloo** [halimkun/blogspot-url-grabber#12] [Q&A] [answered]
saya lihat hanya untuk blogsspot, ?
bisakah ini dipakau buat website yang berbasis wordpress atau selain framework seperti itu ? 

misal dipakai dalam web hasil buatan sendri (PHP, HTML, CSSS)

**Comment by halimkun** [comment, marked as answer]
untuk sekarang hanya bisa dipakai untuk blogspot saja,

**Reply by ikiteloo** [reply]
oke, 
untuk kedepannya akan saya tunggu updatenya untuk bisa digunakan di banyak platform

**Comment by ikiteloo** [comment]
saya sudah mencobanya pada blogspot, dan terlihat bagus,
sepertinya akan lebih keren jika bisa digunakan dalam banyak platform, 

karena ini akan mempermudah

**Reply by halimkun** [reply]
Oke @ikiteloo 
kedepanya akan kami siapkan untuk bisa lebih fleksibel dan dipergunakan dengan baik

terimakasih untuk tanggapan baikya

**Reply by ikiteloo** [reply]
oke, saya mengunggu hal baikknya.

**Reply by ikiteloo** [reply]
apakah anda pakai streamlite untuk membuatnya bisa diakses dari browser
